core/input/input_map.cpp

```diff
@@ -157,7 +157,7 @@ List<StringName> InputMap::get_actions() const {
 }
 
 List<Ref<InputEvent>>::Element *InputMap::_find_event(Action &p_action, const Ref<InputEvent> &p_event, bool p_exact_match, bool *r_pressed, float *r_strength, float *r_raw_strength, int *r_event_index) const {
-	ERR_FAIL_COND_V(!p_event.is_valid(), nullptr);
+	ERR_FAIL_COND_V(p_event.is_null(), nullptr);
```

Comment:
Surely in another PR, but I could've sworn there's an `ERR_FAIL_NULL` or similar for these situations.

Reply:
Yes but we actively don't use `ERR_FAIL_NULL` for `Ref`, there are some cases violating this rule but most don't (and planning on doing some cleanup for those), we prefer using explicit checks for the `is_valid` or `is_null` with `Ref`, partially to make it very obvious it *is* a `Ref` but also because we *might* some time down the line make `is_null` distinct from just a simple null check if we change some internal features in `Ref`